src/grading-settings/grading-scale/GradingScale.jsx

```diff
@@ -144,7 +144,7 @@ const GradingScale = ({
       const updatedLetters = [...prevLetters];
       updatedLetters.splice(updatedLetters.length - 1, 1);
 
-      return updatedLetters.length === 1 ? ['pass'] : updatedLetters;
+      return updatedLetters.length === 1 ? ['Pass'] : updatedLetters;
```

Comment:
Hardcode best practices from legacy...
It'd be nice to extract this label as translatable constant.

Reply:
Yes, your are right 👍